### ISSUE TITLE
Tests fail intermittently because they don't succeed quickly enough with flaky workers.

### ISSUE THREAD

**Issue by adamnovak** [DataBiosphere/toil#2925]
See for example https://ucsc-ci.com/databiosphere/toil/-/jobs/35820

```
__________________ JobServiceTest.testServiceWithCheckpoints ___________________
Traceback (most recent call last):
  File "/builds/databiosphere/toil/src/toil/test/src/jobServiceTest.py", line 181, in runToil
    Job.Runner.startToil(rootJob, options)
  File "/builds/databiosphere/toil/src/toil/job.py", line 793, in startToil
    return toil.restart()
  File "/builds/databiosphere/toil/src/toil/common.py", line 832, in restart
    return self._runMainLoop(rootJobGraph)
  File "/builds/databiosphere/toil/src/toil/common.py", line 1070, in _runMainLoop
    jobCache=self._jobCache).run()
  File "/builds/databiosphere/toil/src/toil/leader.py", line 246, in run
    raise FailedJobsException(self.config.jobStore, self.toilState.totalFailedJobs, self.jobStore)
toil.leader.FailedJobsException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/builds/databiosphere/toil/src/toil/test/src/jobServiceTest.py", line 116, in testServiceWithCheckpoints
    self.testService(checkpoint=True)
  File "/builds/databiosphere/toil/src/toil/test/src/jobServiceTest.py", line 73, in testService
    self.runToil(t)
  File "/builds/databiosphere/toil/src/toil/test/src/jobServiceTest.py", line 186, in runToil
    self.fail() #Exceeded a reasonable number of restarts
  File "/usr/lib/python3.6/unittest/case.py", line 670, in fail
    raise self.failureException(msg)
AssertionError: None
```

This has caused problems for our PRs as in https://github.com/DataBiosphere/toil/pull/2904#issuecomment-575815997

We should reduce the worker failure rate or increase the number of restarts we allow, so that the test passes with much higher probability.

┆Issue is synchronized with this [Jira Task](https://ucsc-cgl.atlassian.net/browse/TOIL-490)
┆Issue Number: TOIL-490

**Comment by adamnovak** [comment]
@jeffrey856 I bet you could tweak this. You just have to find the worker failure rate or restart count for this test and change one or both of them.

**Comment by jeffrey856** [comment]
Fixed by #2936
